plugin/ale/aaflags.vim

```diff
@@ -39,3 +39,11 @@ let g:ale_warn_about_trailing_whitespace =
 
 " This flag can be set to 1 to keep sign gutter always open
 let g:ale_sign_column_always = get(g:, 'ale_sign_column_always', 0)
+
+" String format for statusline
+" Its a list where:
+" * The 1st element is for errors
+" * The 2nd element is for warnings
+let g:ale_statusline_format = get(
+			\	g:, 'ale_statusline_format', ['%d error(s)', '%d warning(s)']
+			\ )
```

Comment:
I'd use a hanging indent here, as mentioned below. You could probably leave the first two arguments to `get` on the first line, and then up-indent the value on the second line, leaving the closing parenthesis on its own line. Like so:

```vim
let g:ale_statusline_format = get(g:, 'ale_statusline_format', 
\   ['%d error(s)', '%d warning(s)']
\)
```